src/Components/Atoms/FAFormControlLabel/FAFormControlLabel.interfaces.tsx

```diff
@@ -0,0 +1,7 @@
+import { FormControlLabelProps } from "@mui/material";
+
+export interface IFAFormControlLabel extends FormControlLabelProps {
+  testID: `ControlLabel-${string}`;
+  control: React.ReactElement<any, any>;
```

Comment:
I didn't see the control props used in FAFormControlLabel component. did linter cover this issue?

Reply:
Miss the line, yet still working just fine. probably because the props are already contributed from material ui props. still, to be cautious I added the props into the line.